packages/gatsby-cli/src/reporter/loggers/ink/context.tsx

```diff
@@ -17,7 +17,7 @@ export const StoreStateProvider: React.FC = ({
     (getStore().getState() as any) as IGatsbyState
   )
 
-  useEffect(
+  useLayoutEffect(
```

Comment:
This seems needed for quick commands (like `gatsby options`) that are in practice short sync ink@3 doesn't seem to run `useEffect` before we start printing things and result in no output in those cases